README.md

```diff
@@ -26,69 +26,70 @@ Install the Docker generator:
 npm install -g generator-docker
 ```
 
-Run the generator in the same folder that your project is in:
+## Getting Started
+Once yo docker and VS Code are installed, here's the few steps to debug your first app
+### Node.js
+- From the root of your project, where your server.js file existts, open a terminal window
+```bash
+$ yo docker
+```
+- Answer the prompts, including your image name
+- Open the file you wish to debug and set a breakpoint
+- From VS Code, **[Command]** + **[P]** for the command palate
```

Comment:
For windows it is Ctrl+P (assuming default key bindings): [https://code.visualstudio.com/docs/customization/keybindings](https://code.visualstudio.com/docs/customization/keybindings)

---
In reply to: [68331884](https://github.com/Microsoft/generator-docker/pull/90#discussion_r68331884) [](ancestors = 68331884)